RMSD.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
```

Comment:
You should have a brief comment up here about what this script is for, you can see the other scripts in this directory for examples. In this section you should also have a list of authors, the same as you would on any thing else we publish.